src/functions-templates/typescript/scheduled-function/{{name}}.ts

```diff
@@ -0,0 +1,15 @@
+import { Handler, schedule } from '@netlify/functions';
+
+const formatAsDateTime = (date: Date) => `${date.toLocaleDateString()} ${date.toLocaleTimeString()}`;
+
+const handler: Handler = () => {
+    const currentExecutionTime = formatAsDateTime(new Date(Date.now()));
```

Comment:
The `new Date()` constructor uses the current date by default, there's no need to supply `Date.now()`.

```suggestion
    const currentExecutionTime = formatAsDateTime(new Date());
```